.github/labels.yml

```
@@ -0,0 +1,161 @@
+# Cross-track labels
+- name: "x:type/docs"
+  description: "Work on Documentation"
+  color: "604fcd"
+
+- name: "x:type/ci"
+  description: "Work on Continuous Integration (e.g. GitHub Actions workflows)"
+  color: "604fcd"
+
+- name: "x:type/docker"
+  description: "Work on Dockerfiles"
+  color: "604fcd"
+
+- name: "x:type/coding"
+  description: "Work on code that will run in production"
+  color: "604fcd"
+
+- name: "x:type/content"
+  description: "Work on content (e.g. exercises, concepts)"
+  color: "604fcd"
+
+- name: "x:action/create"
+  description: "Work on something from scratch"
+  color: "6f60d2"
+
+- name: "x:action/sync"
+  description: "Sync content with its latest version"
+  color: "6f60d2"
+
+- name: "x:action/improve"
+  description: "Improve existing functionality/content"
+  color: "6f60d2"
+
+- name: "x:action/proofread"
+  description: "Proofread text"
+  color: "6f60d2"
+
+- name: "x:action/fix"
+  description: "Fix an issue"
+  color: "6f60d2"
+
+- name: "x:size/xs"
+  description: "Trivial amount of work"
+  color: "5240c9"
+
+- name: "x:size/s"
+  description: "Small amount of work"
+  color: "5240c9"
+
+- name: "x:size/m"
+  description: "Average amount of work"
+  color: "5240c9"
+
+- name: "x:size/l"
+  description: "Large amount of work"
+  color: "5240c9"
+
+- name: "x:size/xl"
+  description: "Massive amount of work"
+  color: "5240c9"
+
+- name: "x:knowledge/newbie"
+  description: "No existing Exercism knowledge required"
+  color: "4836bf"
+
+- name: "x:knowledge/beginner"
```

```suggestion
- name: "x:knowledge/rudimentary"
```

(Possibly `basic` instead if non-native speakers aren't familiar with rudimentary)